warehouse/query-core/src/main/java/datawave/query/tables/content/ContentQueryTable.java

```diff
@@ -55,7 +55,9 @@ public class ContentQueryTable extends BaseQueryLogic<Entry<Key,Value>> {
     private static final String ALL = "\u10FFFF";
 
     private int queryThreads = 100;
-    private ScannerFactory scannerFactory;
+
+    ScannerFactory scannerFactory;
+
     private String viewName = null;
```

Comment:
This could also be package private if it's only used for tests.

Reply:
rgr, fixing.